addons/web/static/src/views/form/form_controller.js

```diff
@@ -255,6 +257,10 @@ export class FormController extends Component {
         }
     }
 
+    async onSavedRecord(record) {}
+
+    async onWillSaveRecord(record) {}
```

Comment:
these two could be documented (explain why they exist, when they are executed and what should be their API if they are overriden)